.eslintrc.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "airbnb-base",
   "rules": {
+    "comma-dangle": 1,
```

Comment:
This does end up emitting a warning, still.  Could be set to zero to ignore warnings if no one really cares.